content/docs/reference/config.mdx

```diff
@@ -55,4 +55,39 @@ export default defineConfig({
 ```
 
 In this example, the TinaCMS admin will be viewable at `<my-site-url>/admin/index.html`.
-For more information [check out the content modeling section](/docs/schema/).
+For more information [check out the content modeling section](/docs/schema/).\
+\
+
+
+## Typescript Path Alias 
+
+TinaCMS supports typeScript path aliases in the tina.config.js file and collections, making it easier to organize and maintain your code.
+
+##### Use alias in config.ts
+
+Ensure that tsconfig.json exists at the root of your project. Define your aliases in the compilerOptions section:
```

Comment:
**consistency**: The introductory paragraph has the files and in-code declaration formatted as in-line code.

```suggestion
Ensure that `tsconfig.json` exists at the root of your project. Define your aliases in the `compilerOptions` section:
```